src/pages/Landing.tsx

```diff
@@ -73,7 +73,7 @@ const Landing = () => {
             <dt>Recent improvements</dt>
             <dd>
               <OffsiteLink
-                to="https://github.com/PhilanthropyDataCommons/data-viewer/blob/main/RECENT_CHANGES.md"
+                to="https://github.com/PhilanthropyDataCommons/front-end/blob/main/RECENT_CHANGES.md"
```

Comment:
I just tested this by clicking through the deploy preview and the new URL is now live.

Reply:
Sigh how I wish we could wildcard the auth domain. :/

Reply:
Me, too. Someday...

Actually, this might be worth re-evaluating separately - it might be worthwhile to do some legwork to enable deploy preview logins automatedly! It won't be out-of-the-box, but I think it should be possible.